src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Pages/Shared/_Layout.cshtml

```diff
@@ -59,7 +59,7 @@
         <script src="~/js/site.js" asp-append-version="true"></script>
     </environment>
     <environment exclude="Development">
-        <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-2.2.0.min.js"
+        <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-3.3.1.min.js"
```

Comment:
Do we need to update the version of jQuery as well?

Reply:
We updated the fallback jQuery libraries already. https://github.com/aspnet/templating/pull/307

Reply:
Oh we actually made that the required minimum? I completely forgot. Looks good then.